Max/System/Tower.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace Version3D {
+    public class Tower : MonoBehaviour {
+
+        public delegate void ScorePoints(int score);
+        public static event ScorePoints SendScore;
+
+        public delegate void BlockFall();
+        public static event BlockFall OnBlockFall;
+
+        private static Tower instance = null;
+        public static Tower Instance
+        {
+            get {
+                if (instance == null) {
+                    // This is where the magic happens.
+                    //  FindObjectOfType(...) returns the first Tower object in the scene.
+                    instance = FindObjectOfType(typeof(Tower)) as Tower;
```

Comment:
Ik snap niet waarom je twee keer checkt of instance null is.
Aangezien je instance variable static is en dus bij elke 'kopie' dezelfde waarde zal hebben. Zou er geen Tower in de scene mogen zijn als instance null is. Hopelijk snap je wat ik bedoel.